napari/settings/_tests/test_settings.py

```diff
@@ -21,7 +21,8 @@ class Config:
             env_prefix = 'testnapari_'
 
     return TestSettings(
-        tmp_path / 'test_settings.yml', schema_version=CURRENT_SCHEMA_VERSION
+        tmp_path / '0.4.19' / 'test_settings.yml',
```

Comment:
Similar comment to https://github.com/napari/napari/pull/5306/files#r1015643699

Reply:
There needs to be a parsable <current version>. Could change this to `1.2.3`.
The new tests use monkey patch on the version so could make those `1.2.3`, `1.2.1` too probably.